### ISSUE TITLE
Plus editor stalls then blank app

### ISSUE THREAD

**Issue by Thenorneron** [standardnotes/forum#676]
Hi, I'm writing my work in Plus editor both on Mac and Windows and have now resorted to web but same thing is cross platform. When I write text, input is kinda slow and jittery and after a while app freezes completely (not unresponsive) just whole app screen is white and I have to force quit and restart and usually last words are missing. This has happened about 30 times now but I need to continue editing... I have tried on Brave browser and after a while has issue and I have to reload. 
There is in plus editor: 
Words 147901 Paragraphs 38336 Characters 2330141 Read Time 740 minutes With very little formatting... 
What is the issue here?

**Comment by mobitar** [comment]
38336 paragraphs? Honestly sounds like it's just too long of a note. Performance here will be subjective. My recommendation is to trim down the size of the note to something your system or browser can handle.

**Comment by Thenorneron** [comment]
Yes that seemed little excessive. Just pasted all text into Plain Editor and now I get this: Words 2247 Paragraphs 97 Characters 16851 Read Time 11 minutes
I don't know what the issue is with Plus Editor? How can I fix it and keep my font sizes, colors, background colors? 
I really really like Standard Notes and focus on privacy and extensions but this kind of issues are off putting and I don't know what to do when similar issues comes up.

**Comment by mobitar** [comment]
Hmm..it might have been the raw HTML markup. Though 2 million characters still sounds off. Perhaps there was some formatting loop that triggered erroneous formatting characters? I'd try using the Plus/Bold editor again and apply your formatting. I don't believe it should balloon to that many characters but if it does, there might be something going on.
